### PR TITLE
Mapathon section and linked to the API

### PR DESCRIPTION
**Pull request by dhakelila**
This PR creates the new Mapathon section. 
Now both "campaigns" and "mapathon" sections are linked to the API.

### CODE REVIEW

File: app/controllers/static_controller.rb [hunk 1]
```diff
@@ -7,6 +7,9 @@ def home
   def campaigns
   end
 
+  def mapathon
```
Comment (simaob) [inline app/controllers/static_controller.rb]: we should follow the convention and use the plural form here as well. "mapathons". And as the entity is actually called "Events" in our database, I'd rename this to "events".

File: app/views/shared/_header.html.erb [hunk 2]
```diff
@@ -26,7 +26,7 @@
           <li class="nav-item sub-menu-toggler"><a href="#" class="nav-link text text-menu -light">Participate</a>
             <ul class="sub-menu">
               <li><%= link_to 'Campaigns', campaigns_path, {class: 'text text-menu -dark'} %></li>
-              <li><%= link_to 'Mapathons', "/", {class: 'text text-menu -dark'} %></li>
+              <li><%= link_to 'Mapathons', mapathon_path, {class: 'text text-menu -dark'} %></li>
```
Comment (simaob) [inline app/views/shared/_header.html.erb]: similar to the previous comment, the path name should be events_path (with event in the plural)

File: app/views/static/mapathon.html.erb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<div id="mapathon">
```
Comment (simaob) [inline app/views/static/mapathon.html.erb]: this file name would need to be updated to match above comment.

File: config/database.yml [hunk 4]
```diff
@@ -20,8 +20,8 @@ default: &default
   # For details on connection pooling, see rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  username: postgres
-  password: postgres
+  username:
```
Comment (simaob) [inline config/database.yml:23]: need to remove this file from the repo. but I'll handle it when merging the PR.

File: config/routes.rb [hunk 5]
```diff
@@ -22,5 +22,6 @@
   end
 
   get '/campaigns', to: 'static#campaigns'
+  get '/mapathon', to: 'static#mapathon'
```
Comment (simaob) [inline config/routes.rb]: same as above, use plural form as it's getting a "collection" of mapathons, and use "events" as it's more generic.

**Comment by simaob** [comment]
@dhakelila I'll take care of the renaming, no need to do it yourself as it's a small change. just thought I'd comment anyway for your information.

**Comment by dhakelila** [comment]
Thanks! I was thinking about ask you that and then I forget. 
Thanks you for take care of the renaming!

**Pull request merged**